commands/station.js

```diff
@@ -50,7 +51,7 @@ export const station = async ({ json, experimental }) => {
   })
 
   const modules = [
-    zinniaRuntime.start({
+    pRetry(() => zinniaRuntime.start({
```

Comment:
With this new implementation, I think the name `start` is no longer accurate. What do you think about renaming `start` to `run`?

```suggestion
    pRetry(() => zinniaRuntime.run({
```

Reply:
Good call, yes!